src/components/molecules/news-and-events-title/news-and-events-title.scss

```
@@ -0,0 +1,21 @@
+@use "../../variables" as *;
+
+.news-event-title{
+    align-items: center;
+    display: flex;
+    margin-bottom: 32px;
+    .news-title{
+        font-size: $fontsize-L;
+        font-family: $fontfamily-mincho;
+        margin-bottom: 0px;
+    }
+    .news-subtitle{
+        font-size: $fontsize-M;
+        margin-left: 16px;
+    }
+    @media(min-width:$breakpoint-lg){
+        .news-title{
+            font-size: $fontsize-XXL;
+        }
+    }
```

```suggestion
.news-event-title{
    align-items: center;
    display: flex;
    margin-bottom: 32px;
    .news-title{
        font-size: $fontsize-L;
        font-family: $fontfamily-mincho;
        margin-bottom: 0px;
        @media(min-width:$breakpoint-lg){
          font-size: $fontsize-XXL;
      }
    }
    .news-subtitle{
        font-size: $fontsize-M;
        margin-left: 16px;
    }
```
こうするとよりシンプルにかけるかと思います。